src/eko/io/runcards.py

```diff
@@ -176,32 +208,49 @@ def targetgrid(self) -> interpolation.XGrid:
     def targetgrid(self, value: interpolation.XGrid):
         self._targetgrid = value
 
+    @classmethod
+    def from_dict(cls, dictionary: dict):
+        """Deserialize rotation.
+
+        Load from full state, but with public names.
+
+        """
+        d = dictionary.copy()
+        for f in fields(cls):
+            if f.name.startswith("_"):
+                d[f.name] = d.pop(f.name[1:])
+        return cls._from_dict(d)
+
+    @property
+    def raw(self):
+        """Serialize rotation.
+
+        Pass through interfaces, access internal values but with a public name.
+
+        """
+        d = self._raw()
+        for key in d.copy():
+            if key.startswith("_"):
+                d[key[1:]] = d.pop(key)
+
+        return d
+
 
 @dataclass
 class OperatorCard(DictLike):
     """Operator Card info."""
 
     mu0: float
     """Initial scale."""
+    mugrid: List[Target]
+    xgrid: interpolation.XGrid
+    """Momentum fraction internal grid."""
```

Comment:
```suggestion
    """Internal momentum fraction grid."""
```

Reply:
Done in 3a380883cfe68afb46ba1dcf25baf5611860e129